CONTRIBUTING.md

```diff
@@ -73,17 +72,17 @@ In the description of your PR include a link to the PR to [Keycloak Documentatio
 ### Picking correct branch for your PR
 
 Quickstarts are organized into the following branches:
-* `main` contains the latest state of the development on par with latest Keycloak.
+* `master` contains the latest state of the development on par with latest Keycloak.
   This is destination for your PR.
 * `latest` is the branch that corresponds to the latest _released_ version of Keycloak.
   You should *not* open PRs to this branch unless there is proper justification.
 
 ### Submitting your PR
 
-When preparing your PR make sure you have a single commit and your branch is rebased on the main branch from the
+When preparing your PR make sure you have a single commit and your branch is rebased on the master branch from the
```

Comment:
```suggestion
When preparing your PR make sure you have a single commit and your branch is rebased on the main branch from the
```

Reply:
That's a very good catch, thank you!

Created a separate PR to fix the master --> main rename: https://github.com/keycloak/keycloak-quickstarts/pull/269